docs/testplan/account-changes-testplan.md

```diff
@@ -0,0 +1,33 @@
+|                                      | 11. Backend behaviour.                                             | TBD.  Will be created chore/command to delete users and their data. User can't login to deleted account.                                                                                                                                                                                                                                                                  |          |
+| Uplink.                              | 12. Uplink after account freezing.                                 | User can't upload/download files after trial  expiration.                                                                                                                                                                                                                                                                                                   |          |
+| Linksharing.                         | 13. Linksharing after trial expiration.                            | User can't share files after trial expiration. Previously created links shouldn't work.                                                                                                                                                                    |          |
+| Partners.                            | 14. IX-Systems free tier users/ Pro users.                         | Only free tier users should be affected.                                                                                                                                                                                                                                                                                                                            |          |
+|                                      | 15. GB-Labs free tier users/ Pro users/ GB-labs packet users.      | Only free tier users should be affected.                                                                                                   |          |
+| Emails.                              | 16. Emails flow for free trial users.                              | TBD. Probably we should duplicate warnings by email.                                                                                                                                                                                                                                                                                                               |          |
```

Comment:
Expected email behavior (may not be testable in the first release with this functionality):
* reminder email 3 days before expiration (should be based on same config as warning in UI)
* notification email when user becomes expired

Reply:
any email about changing plan? like "Inform you about changing account plan from 'Free tier' to 'Free trial' ..."

Reply:
These emails should only occur for users who "naturally" enter free trial. E.g.  if they created an account after config=30days
Users affected by the "free tier" -> "free trial" migration should _not_ receive emails from the satellite. These email reminders are being handled manually